test/bench/ok/bignum.drun-run.ok

```diff
@@ -1,6 +1,6 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
-debug.print: {cycles = 2_508_947; size = +59_652}
+debug.print: {cycles = 2_594_922; size = +59_652}
 ingress Completed: Reply: 0x4449444c0000
-debug.print: {cycles = 105_771_809; size = +1_817_872}
+debug.print: {cycles = 111_509_014; size = +1_817_872}
```

Comment:
I analyzed it and the problem is that the Rust/LLVM builtin `memcpy` is way slower than `musl`, not using 64-bit copying. I activated bulk memory copying to get better performance.
(Bignum needs a lot of `memcpy` because of big integer reallocations to larger blobs.)